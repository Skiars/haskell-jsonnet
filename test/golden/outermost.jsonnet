local test = [
  {
    foo: "test1",
    bar: {
      baz: {
        qux: {
          quux: $.foo
        }
      }
    }
  },
  {
    foo: "test2",
    bar: {
      baz: {
        qux: {
          quux: $["foo"]
        }
      }
    }
  }
];

test
