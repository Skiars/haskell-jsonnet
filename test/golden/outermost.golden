[
  {
    "foo": "test1",
    "bar": {"baz": {"qux": {"quux": "test1"}}}
  },
  {
    "foo": "test2",
    "bar": {"baz": {"qux": {"quux": "test2"}}}
  }
]